### PR TITLE
[#911] Added configuration options and logic to hide unused functionality

### PR DESCRIPTION
**Pull request by Bartvaderkin**
We had to do some interpretation on the ticket, because Actions are a thing with or without Plan/Samenwerken. 

So I made it so there is an option to disable Samenwerken and remove it from menu but keep general Action functionality, and another option to also disable the general Actions (which requires disabled Plan).

Additionally there was an ask to hide QuestionaireStep (Zelfcheck) if there are no published QuestionaireStep's, which was more of a bug-fix then a feature request.

It might make sense to overview the code per-commit as they are reasonably clean.

### CODE REVIEW

File: src/open_inwoner/configurations/models.py [hunk 1]
```diff
@@ -354,6 +369,13 @@ class Meta:
     def __str__(self):
         return str(_("Site Configuration"))
 
+    def clean(self):
+        super().clean()
+
+        if self.show_plans and not self.show_actions:
```
Comment (alextreme) [inline src/open_inwoner/configurations/models.py:375]: Netjes opgelost :)

**Review by alextreme — APPROVED** [review]

**Comment by alextreme** [comment]
@vaszig thanks, glad you double-checked :+1:

**Comment by Bartvaderkin** [comment]
@vaszig I don't see it here but I'm still interested about the migration feedback, but anyway, there are two because I was trying to make clean commits instead of piling everything into one.

**Comment by vaszig** [comment]
> @vaszig I don't see it here but I'm still interested about the migration feedback, but anyway, there are two because I was trying to make clean commits instead of piling everything into one.

Yes, it's clear enough. It makes sense to have them separately according to the nice and clear functionality-based commits.

**Pull request merged**